src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/ExplainIT.java

```diff
@@ -211,4 +212,37 @@ public void testContentTypeOfExplainRequestShouldBeJson() throws IOException {
 
         assertEquals("application/json; charset=UTF-8", response.getHeader("content-type"));
     }
+
+    @Test
+    public void assertExplainPrettyFormatted() throws IOException {
+        String query = StringUtils.format("SELECT firstname FROM %s", TEST_INDEX_ACCOUNT);
+
+        String notPrettyExplainOutputFilePath = TestUtils.getResourceFilePath(
+                "src/test/resources/expectedOutput/explainIT_format_not_pretty.json");
+        String notPrettyExplainOutput = Files.toString(new File(notPrettyExplainOutputFilePath), StandardCharsets.UTF_8);
+
+        assertThat(executeExplainRequest(query, ""), equalTo(notPrettyExplainOutput));
+        assertThat(executeExplainRequest(query, "pretty=false"), equalTo(notPrettyExplainOutput));
+
+        String prettyExplainOutputFilePath = TestUtils.getResourceFilePath(
```

Comment:
Please extract that part to different test

Reply:
Done